common/dal-context/src/context.rs

```diff
@@ -22,9 +22,9 @@ use opendal::ops::OpStat;
 use opendal::ops::OpWrite;
 use opendal::readers::CallbackReader;
 use opendal::Accessor;
+use opendal::BoxedAsyncRead;
```

Comment:
LGTM.
A small question:
`BoxedAsyncRead` is `BoxedAsyncReader` or a `Read` opeartion?

Reply:
`BoxedAsyncRead` is the same with `Box<dyn AsyncRead>`

Reply:
Feel that `BoxedAsyncReader` is easier to understand as a reader, `BoxedAsyncRead` like a reading action :)

Reply:
LGTM, let's do this in opendal `v0.0.6`!

Reply:
Tracked in https://github.com/datafuselabs/opendal/issues/52